src/backend/langflow/components/llms/AzureChatOpenAI.py

```diff
@@ -61,16 +61,16 @@ def build_config(self):
     def build(
         self,
         model: str,
-        azure_endpoint: str,
+        openai_api_base: str,
         azure_deployment: str,
         api_key: str,
-        api_version: str = "2023-05-15",
```

Comment:
Hey @lucifertrj 

Thanks for the PR.

I think there's a mistake here. This will make so the field displays the word `"str"`.